pkg/templates/nstemplatetiers/testtemplates/fakenstemplatetiers/test/cluster.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: template.openshift.io/v1
```

Comment:
minor - do we need to have this nested folders or could it just be :
`testtemplates/testtier/cluster|ns_dev|spacerole_admin|tier.yaml` 

I mean removing the `fakenstemplatetier` folder.

Reply:
so the logic is such that in it does require 4 step folder structure as it in deploy/ for actual prod . thats how we are getting the path. and hence this structure requires `faketesttemplatetier` . so for test cases we need to have same folder structure. 
https://github.com/codeready-toolchain/host-operator/pull/1165/files#diff-75fb59490e920da60f86dae8e59891d725272ef82d0515a32447036e0b576e40R44

Reply:
yeah, it's a bit weird but not sure how we can make it more flexible.